css/style.css

```diff
@@ -0,0 +1,51 @@
+body {
+  background-color: #000;
+  display: flex;
+  flex-direction: column;
+}
+
+h1 {
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 56px;
+  line-height: 64px;
+  letter-spacing: 0.37px;
+  color: #ff6b00;
+}
+
+h3 {
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 24px;
+  line-height: 32px;
+  color: #fff5e1;
+}
+
+p {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+}
+
+ul {
+  list-style-type: none;
+}
```

Comment:
Kindly add the following styling to ul class this is for your social media icons 

ul {
    list-style-type: none;
    gap: 20px;
    padding: 1px;
}

Implement all the changes requested and your page will look like this:
![Screenshot from 2022-04-21 19-42-00](https://user-images.githubusercontent.com/79506105/164499163-bae38b65-bae7-4364-a296-9cec010d0c43.png)
 This is how your design looks:
![Screenshot from 2022-04-21 19-45-13](https://user-images.githubusercontent.com/79506105/164499448-700ee7e4-1d66-49ee-af4d-687a891c8a24.png)